data/json/items/tool_armor.json

```
@@ -2518,5 +2518,52 @@
     "coverage": 25,
     "material_thickness": 4,
     "use_action": "DIVE_TANK"
+  },
+  {
+    "id": "electric_blanket",
+    "type": "TOOL_ARMOR",
+    "name": "electric blanket",
+    "description": "A heated blanket made of polyster.  It's the most comfortable thing to ever grace the Earth.  It's usually meant to be plugged into an outlet, but the battery compartment will have to do.",
```

```suggestion
    "description": "A heated blanket made of polyester.  There's nothing cozier for long, cold nights.  It's usually meant to be plugged into an outlet, but the battery compartment will have to do.",
```

---

Is it too informal to have it joke about being the most comfortable thing?

---

No, it's just a bit jarring to me in a world consisting of super morale boosting Toast 'Ems, all kinds of drugs, mega inspiring books and Marloss Berries.

Maybe in a situational sentence, it would sound better?

```suggestion
    "description": "A heated blanket made of polyester.  It's the most comfortable thing in the world with a good book, reading light and pillow fort.  It's usually meant to be plugged into an outlet, but the battery compartment will have to do.",
```